### DISCUSSION TITLE
Unexpected TemplateToken type `3`

### DISCUSSION THREAD

**Discussion by DecoyRS** [community/community#25321] [Actions]
<p>I’ve switched default branch in my GitHub repo and changed workflow to trigger for changes for new branch, but actions for new default branch will fail with message <code>The workflow is not valid. Unexpected TemplateToken type '3'</code> on the first line of workflow file.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/bf7c7d303cc18da00d790504b364f7218dadc124.png" data-download-href="/uploads/short-url/rjXUCTK6aEk06bfK2EgNVCee3zK.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1694911/181090428-034d89b4-fcd4-4c09-b9b8-91440da81dea.png" alt="image" data-base62-sha1="rjXUCTK6aEk06bfK2EgNVCee3zK" width="690" height="160" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/bf7c7d303cc18da00d790504b364f7218dadc124_2_690x160.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/bf7c7d303cc18da00d790504b364f7218dadc124_2_1035x240.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/b/bf7c7d303cc18da00d790504b364f7218dadc124_2_1380x320.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/bf7c7d303cc18da00d790504b364f7218dadc124_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1504×349 14.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
Changing workflow back to be trigger on old branch works fine.<br>
UPD: here’s the link to my actions history: <a href="https://github.com/JetBrains/UnrealLink/actions" rel="nofollow noopener">https://github.com/JetBrains/UnrealLink/actions</a>
<p>SOLVED: I’ve used ${{ github.run_number }} in matrix field. The error is extremely missleading.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<aside class="quote no-group" data-username="DecoyRS" data-post="1" data-topic="119375">
<div class="title">
<div class="quote-controls"></div>
 DecoyRS:</div>
<blockquote>
<p>SOLVED: I’ve used ${{ github.run_number }} in matrix field. The error is extremely missleading.</p>
</blockquote>
</aside>
<p>Glad that you found your own solution to this. I’m highlighting this in case anyone else stumbles across your question and has the same issue. Thank you for sharing it with us.</p>

**Comment by ghost** [comment]
<p>Hi – sorry you’ve experienced this.</p>
<p>Just wanted to let you know we do have a fix in place for this. Although we do not have an ETA on when it might be deployed, we hope it will be in the next couple of weeks. Thank you for your patience!</p>
